tests/storage/snapshot.go

```diff
@@ -1180,6 +1132,89 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 			})
 		})
 
+		Context("With more complicated VM with/out GC of succeeded DV", func() {
```

Comment:
Same `[serial]` might be needed as it could cause other tests to fail otherwise.

Reply:
adding